nemo_rl/utils/checkpoint.py

```
@@ -265,6 +271,14 @@ def get_latest_checkpoint_path(self) -> Optional[str]:
         step_dirs.sort(key=lambda x: int(Path(x).name.split("_")[1]))
         if len(step_dirs) == 0:
             return None
+        # Load the config.yaml from this ckpt dir
+        config_path = os.path.join(step_dirs[-1], "config.yaml")
+        if os.path.exists(config_path):
+            ckpt_config = load_config(config_path)
+            if ckpt_config["policy"]["model_name"] != self.model_name:
+                raise ValueError(
+                    f"Model name in checkpoint {ckpt_config['policy']['model_name']} does not match policy model name {self.model_name}. Please verify if the correct checkpoint is being loaded."
+                )
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)